src/Compilers/VisualBasic/Test/Emit/Emit/EditAndContinueTests.vb

```diff
@@ -3325,7 +3326,7 @@ End Namespace
                                                                      testData0.GetMethodData("M.B.M").EncDebugInfoProvider)
                 Dim method0 = compilation0.GetMember(Of MethodSymbol)("M.B.M")
                 Dim reader0 = md0.MetadataReader
-                CheckNames(reader0, reader0.GetTypeDefNames(), "<Module>", "VB$AnonymousType_1`2", "VB$AnonymousType_0`2", "VB$AnonymousType_2`1", "A", "B")
+                CheckNames(reader0, reader0.GetTypeDefNames(), "<Module>", "VB$AnonymousType_1`2", "VB$AnonymousType_2`1", "VB$AnonymousType_0`2", "A", "B")
```

Comment:
Yes, I field the issue. :)

In the meantime we can 1) disable the test or 2) pick an architecture that it will succeed on and only run tests in that arch. That arch is amd64. :)